packages/prisma/migrations/20231027144522_add_ical_columns_to_booking/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Added the required column `iCalUID` to the `Booking` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "iCalSequence" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "iCalUID" TEXT NOT NULL;
```

Comment:
We should use VARCHAR here because TEXT columns are historically known to be worse in terms of storage and perf. Maybe there's a max we know this will be? 255?